azext_connect/custom.py

```diff
@@ -483,38 +486,18 @@ def bind_function(
         logger.error(e)
 
 
-def _validate(
-    cmd, subscription, resource_group, name, source, target, authtype, permission=None, client_id=None,
-    client_secret=None, username=None, password=None, additional_info={}
-):
-    if not AuthType.has_value(authtype):
-        raise Exception('Auth type not supported')
-    auth_info = AuthInfo(
-        AuthType(authtype), permission, client_id, client_secret, username, password
-    )
-    api = _create_api(cmd)
-    result = api.validate(subscription, resource_group, name, source, target, auth_info, additional_info)
-    if result.ok is not True:
-        end = result.text.find('\r\n\r\nHEADERS\r\n=======')
-        msg = result.text[:end] if end > -1 else result.text
-        err_msg = 'Fail to validate {0} with {1} connection. Code:{2}. Detail:{3}'.format(source, target, result.status_code, msg)
-        raise Exception(err_msg)
-    res_obj = json.loads(result.text)
-    return res_obj
-
-
-def validate_springcloud(
-    cmd, resource_group, name, springcloud, appname, mysql=None, database=None, username=None, password=None
-):
+def validate_general(cmd, resource_group, name):
     try:
         subscription = get_subscription_id(cmd.cli_ctx)
-        scope = '/subscriptions/{0}/resourceGroups/{1}'.format(subscription, resource_group)
-        source = '{0}/providers/Microsoft.AppPlatform/Spring/{1}/Apps/{2}'.format(scope, springcloud, appname)
-        target = _get_target_id(scope, mysql=mysql, database=database)
-        result = _validate(
-            cmd, subscription, resource_group, name, source, target, authtype='Secret', username=username, password=password
-        )
-        print(json.dumps(result, indent=2))
+        api = _create_api(cmd)
+        result = api.validate(subscription, resource_group, name)
+        if result.ok is not True:
+            end = result.text.find('\r\n\r\nHEADERS\r\n=======')
+            msg = result.text[:end] if end > -1 else result.text
```

Comment:
Why find "HEADER" in the result?
I think we can check the validation result response body if the connection is available

Reply:
This is a workaround for the current error handling message, which out put all the request payload. So just remove the request payload.

Reply:
OK~

Reply:
This part is for when result is not ok which means connection is not available.